cmd/minikube/cmd/stop.go

```diff
@@ -115,8 +115,8 @@ func stopProfile(profile string) int {
 	}
 
 	if !keepActive {
-		if err := kubeconfig.UnsetCurrentContext(profile, kubeconfig.PathFromEnv()); err != nil {
-			exit.Error(reason.HostKubeconfigUnset, "update config", err)
+		if err := kubeconfig.DeleteContext(profile, kubeconfig.PathFromEnv()); err != nil {
+			exit.Error(reason.HostKubeconfigUpdate, "update config", err)
```

Comment:
lets create a new Reason Exit here 
https://github.com/medyagh/minikube/blob/82a5189b934ea02d127be85dbdba50dfa4282984/pkg/minikube/reason/reason.go#L207

```
	HostKubeconfigDelete = Kind{ID: "HOST_KUBECONFIG_DELETE", ExitCode: ExHostConfig}
```

Reply:
@medyagh you are most welcome!
that did cross my mind, but `HostKubeconfigDelete` sounded a bit wrong (we are not going to delete kubeconfig), while `HostKubeconfigUpdate` not so bad as we are effectively just updating the kubeconfig
on the other hand, maybe:
```
	HostKubeconfigContextDelete = Kind{ID: "HOST_KUBECONFIG_CONTEXT_DELETE", ExitCode: ExHostConfig}
```
would be more appropriate?
what do you think?

Reply:
that is actually good thinking ! I like that

how about something shorter but unique

"HostKubeconfigDeleteCtx"

the goal is if users collect these exit codes as metrics, they should find out what was the cause of the problem in a unique word

Reply:
yes, that sounds just about right :)
the update is on its way...
thanks @medyagh !